app/javascript/src/components/Dashboard/Notes/Note/Header.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+import { Typography, Dropdown } from 'neetoui'
+import { MenuVertical } from 'neetoicons'
+
+const Header = ({note, onClickEdit, onClickDelete}) => {
+
+  const handleEdit = (selectedNote) => {
+    onClickEdit(selectedNote);
+  }
+
+  const handleDelete = (selectedNoteId) => {
+    onClickDelete(selectedNoteId);
+  }
```

Comment:
1) These util methods, handleEdit and handleDelete, are an overkill. The props are already well defined. These utils don't add any significant value because they don't perform any other side effect.
2) The prop names should be changed. They are handlers of an `onXXX` event. Name it accordingly.